rdkit/Chem/SATIS.py

```diff
@@ -12,15 +12,12 @@
 
```

Comment:
Doesn't show up here in the diff, but you might as well remove the `$Id$` line here and update the copyright.